app/components/Footer.tsx

```diff
@@ -1,10 +1,165 @@
+        <div>
+          <p className="mb-2 text-lg font-bold">Markets</p>
+          <ul className="space-y-1">
+            {Object.values(Categories).map(category => (
+              <li className="text-md font-medium capitalize" key={category}>
+                <a href={`/?c=${category}`}>
+                  <p className="hover:text-text-primary-main">{category}</p>
+                </a>
+              </li>
+            ))}
+          </ul>
+        </div>
+        <div>
+          <p className="mb-2 text-lg font-bold">Socials</p>
+          <ul className="space-y-1">
+            <li className="text-md capitalize">
+              <a href={X_URL} target="_blank">
+                <p className="hover:text-text-primary-main">X</p>
+              </a>
+            </li>
+            <li className="text-md capitalize">
+              <a href={DISCORD_URL} target="_blank">
+                <p className="hover:text-text-primary-main">Discord</p>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+    <MadeBySwapr />
+  </div>
+);
+
+const PoweredByPartners = () => (
+  <div className="flex flex-col space-y-2 rounded-8 bg-surface-surface-1 p-3 md:flex-row md:items-center md:space-x-2 md:space-y-0">
+    <p className="mb-2 text-nowrap md:mb-0 md:mr-2">Powered by</p>
+    <div className="grid grid-cols-2 gap-2 md:flex">
+      <a
+        href="https://www.gnosis.io/"
+        className="flex items-center space-x-1"
+        target="_blank"
+      >
+        <GnosisIcon />
+        <p className="font-bold">Gnosis</p>
+      </a>
+      <span className="hidden md:inline-block">•</span>
+      <a href="https://reality.eth.limo/" className="font-bold" target="_blank">
+        Reality.eth
+      </a>
+      <span className="hidden md:inline-block">•</span>
+      <a
+        href="https://kleros.io/"
+        className="flex items-center space-x-1"
+        target="_blank"
+      >
+        <KlerosIcon />
+        <p className="font-bold">Kleros</p>
+      </a>
+      <span className="hidden md:inline-block">•</span>
+      <a href="https://www.gnosis.io/labs" className="font-bold" target="_blank">
+        Gnosis AI
+      </a>
+    </div>
+  </div>
+);
+
+const MadeBySwapr = () => (
+  <div className="mt-12 flex items-center justify-center border-t border-surface-surface-2 py-5 md:justify-start">
+    <div className="flex space-x-2">
+      <p>Made with ☕️ by</p>
+      <a
+        href="https://swapr.eth.limo"
+        target="_blank"
+        className="flex items-center space-x-1"
+      >
+        <div>
+          <SwaprIcon />
+        </div>
+        <p className="font-semibold text-text-primary-main">Swapr</p>
       </a>
-    </p>
+    </div>
   </div>
 );
+
+const Logo = () => (
+  <Link href="/">
+    <div className="text-white flex items-center text-[24px] font-black md:space-x-2">
+      <p>👁️</p>
+      <p>{APP_NAME}</p>
+    </div>
+  </Link>
+);
+
+const SwaprIcon = () => (
```

Comment:
I'd move the 3 icons (Swapr, Gnosis and Kleros) to a separate assets folder, and not have them here.

Reply:
Not sure about this, as I we probably will not use them anywhere else. Being on this file I agree it makes more complex but keeps complexity here instead and does not add more files to the project.

@berteotti what's your thoughts on this?

Reply:
Agree on adding complexity in this file only instead of adding complexity to the codebase. If we need this in more than 1 file, let's move it. Otherwise it can live here.